st2actions/st2actions/bootstrap/runnersregistrar.py

```diff
@@ -382,7 +384,8 @@
             'password': {
```

Comment:
Yeah, we already kinda do that - we have a static global blacklist of attribute / variable names which are masked - https://github.com/StackStorm/st2/pull/1615/files#diff-32ea10d4a187e2830fe2faf165473ca6R24